backend/private-graph/graph/schema.graphqls

```diff
@@ -535,6 +535,7 @@ type ErrorGroup {
 	first_occurrence: Timestamp
 	last_occurrence: Timestamp
 	viewed: Boolean
+	serviceName: String
```

Comment:
> I believe this will only get set for new error groups due to the fingerprint logic?

It's not related to the fingerprinting but whether one passes service name through the SDK. 

>i.e. Any existing error group will not be updated when a new instance is created, and it will represent the serviceName of the first instance if multiple services have the same error?

The service name on the error group is based on the last instance received. It happens here: https://github.com/highlight/highlight/pull/6108/files#diff-37a818bb25fd3cee29d865b3306a81501a1371a5ba2e6f1927464f21a56b20a1R386-R416